blossom_wrapper/__init__.py

```diff
@@ -113,35 +113,67 @@ def _call(
 
         return resp
 
-    def get(self, path: str, data=None, params=None) -> Response:
+    def get(self, path: str, data=None, params=None) -> BlossomResponse:
+        """Request a GET request to the API."""
+        response = self._get(path, data, params)
+        if response.status_code == 200:
+            return BlossomResponse(data=response.json())
+        response.raise_for_status()
+        return BlossomResponse()
```

Comment:
Right now, any 200 family response will get condensed into a BlossomStatus.ok -- is that what we want?

Reply:
Looks like it covers a lot more than that, if we were to treat Blossom as if it were some third-party web service.

Any 10x, 20x, or 30x response will elicit a `BlossomStatus.ok` response containing no data. According to [HTTP status code spec](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status), that may not work out too well.

Specifically I'm looking at 204 (no content), 201 (created), and 202 (accepted). The latter two might produce data and we should probably handle for that, even if it's parsing a `{ "message": "y'all are awesome" }` sort of response.

Reply:
The feedback definitely makes sense; how do we want to handle this instead? Do we just always want to return OK for anything that response.raise_for_status does not trigger?

Reply:
My suggestion is to extend the BlossomResponse class to add at a `status_code` field, so if it doesn't error out in the view from `raise_for_status()`, then we can just do

```python
return BlossomResponse(status_code=response.status_code, data=response.json())
```